geoportal/c2cgeoportal_geoportal/views/dynamic.py

```diff
@@ -100,7 +103,8 @@ def _interface(self, interface_config, interface_name, dynamic):
 
     @view_config(route_name="dynamic", renderer="json")
     def dynamic(self):
-        interface_name = self.request.get_organization_interface(self.request.params.get("interface"))
+        route_interface_name = self.request.params.get("interface")
```

Comment:
Seems it might be root_interface_name (interface name before expansion by organization)